sdks/java/extensions/avro/build.gradle

```diff
@@ -35,7 +41,12 @@ def avroVersions = [
     '1111': "1.11.1",
 ]
 
-avroVersions.each { k, v -> configurations.create("avroVersion$k") }
+avroVersions.each { k, v ->
+  configurations {
+    create("avroVersion$k").extendsFrom(implementation)
+    create("avroVersion${k}Generate")
```

Comment:
Create a config for the `"generateAvroClasses$k"` (`avro-tools` and dependencies `avro-compiler` are not required for the tests)